CMakeLists.txt

```diff
@@ -87,21 +87,21 @@ if (VTK_VERSION VERSION_LESS "8.90.0")
 else ()
   # vtk_module_autoinit is needed
   vtk_module_autoinit(
-    TARGETS preciceMap testing
+    TARGETS precice-aste-run testing
     MODULES ${VTK_LIBRARIES}
     )
 endif()
 
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/src/partition_mesh.py      DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/src/join_mesh.py           DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/src/vtk_calculator.py      DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/src/precice-aste-partition.py      DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/src/precice-aste-join.py           DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/src/precice-aste-evaluate.py      DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Do we have to keep the `.py` extension or can we omit it?

Reply:
We can omit them. Probably the best way is to keep them in the source folder but install them without extension.

Those scripts are executed by `/usr/bin/env python3` as long as python3 is installed, I should be safe.

Reply:
>  Probably the best way is to keep them in the source folder but install them without extension.

Why?

Reply:
The developers can easily recognize the files from the extension. Removing the extension from the source file just makes it harder to understand I don't see any benefit. On the other hand, it does not matter whether it is a script file or binary executable for the user. Removing extension makes it closer to the "usual" executable experience.

Reply:
I would not want have differences between the source code version and the build tree version. Let's go without the extension.

Reply:
It is ready to merge.